include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -207,6 +207,33 @@ class __scan_single_wg_kernel;
 template <typename... _Name>
 class __scan_single_wg_dynamic_kernel;
 
+namespace __scan_detail
+{
+template <typename _ValueType, bool _Inclusive, typename _Group, typename _Begin, typename _End,
+          typename _BinaryOperation>
+void
+__work_group_scan(const _Group& __group, _Begin __begin, _End __end, _BinaryOperation __bin_op,
```

Comment:
probably it's better to start the function name with the verb, like scan_work_group

Reply:
Fixed.